ingest/src/server_mobile.rs

```diff
@@ -218,7 +221,15 @@ where
         &self,
         request: Request<CellHeartbeatReqV1>,
     ) -> GrpcResult<CellHeartbeatRespV1> {
-        let timestamp: u64 = Utc::now().timestamp_millis() as u64;
+        let timestamp = Utc::now();
+
+        if timestamp >= self.cbrs_disable_time {
+            return Err(Status::failed_precondition(format!(
+                "CBRS radios are no longer supported as of {}",
```

Comment:
I thought we’d  drop reports not error outfit avoid clients that haven’t updated yet?

Reply:
We did say that when talking through having this code in the verifiers, but when we moved to the ingestor I figured we would just return an error, but I can change to return Ok and then drop on the floor if you prefer.

Reply:
I'm ok erroring out but we'd have to make sure that upstream services can handle gettign an error back wihout either breaking or retrying like crazy